service/src/test/resources/ehr/mapper/diagnosticreport/fhir_bundle.json

```diff
@@ -8,725 +8,802 @@
-              "display":"Comment note"
+              "system": "http://snomed.info/sct",
+              "code": "37331000000100",
+              "display": "Comment note"
             }
           ]
         },
-        "subject":{
-          "reference":"Patient/DAED5527-1985-45D9-993E-C5FF51F36828"
+        "subject": {
+          "reference": "Patient/DAED5527-1985-45D9-993E-C5FF51F36828"
         },
-        "effectiveDateTime":"2020-12-15T15:17:13.51+00:00",
-        "issued":"2020-12-15T15:17:13.51+00:00",
-        "performer":[
+        "effectiveDateTime": "2020-12-15T15:17:13.51+00:00",
+        "issued": "2020-12-15T15:17:13.51+00:00",
+        "performer": [
           {
-            "reference":"Practitioner/C8FD0E2C-3124-4C72-AC8D-ABEA65537D1B"
+            "reference": "Practitioner/C8FD0E2C-3124-4C72-AC8D-ABEA65537D1B"
           }
         ],
-        "comment":"(q) - Normal - No Action",
-        "related":[
+        "comment": "(q) - Normal - No Action",
+        "related": [
           {
-            "type":"derived-from",
-            "target":{
-              "reference":"Observation/D0A358C9-4833-4827-B14B-E8515C25CB12"
+            "type": "derived-from",
+            "target": {
+              "reference": "Observation/D0A358C9-4833-4827-B14B-E8515C25CB12"
             }
           }
         ]
       }
     },
+    {
+      "resource": {
+        "resourceType": "Observation",
+        "id": "TEST-RESULT-NO-SPECIMEN",
+        "meta": {
+          "profile": [
+            "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Observation-1"
+          ]
+        },
+        "identifier": [
+          {
+            "system": "https://wiremock/",
+            "value": "TEST-RESULT-NO-SPECIMEN"
+          }
+        ],
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://read.info/readv2",
+              "code": "42R5.00",
+              "display": "Serum TIBC",
+              "userSelected": true
+            },
+            {
+              "extension": [
+                {
+                  "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
+                  "extension": [
+                    {
+                      "url": "descriptionId",
+                      "valueId": "2580111000000118"
+                    }
+                  ]
+                }
+              ],
+              "system": "http://snomed.info/sct",
+              "code": "1015451000000101",
+              "display": "Serum total iron binding capacity"
+            }
+          ]
+        },
+        "subject": {
+          "reference": "Patient/DAED5527-1985-45D9-993E-C5FF51F36828"
+        },
+        "effectiveDateTime": "2001-02-03T12:00:00+00:00",
+        "issued": "2001-02-03T13:00:00+00:00",
+        "valueQuantity": {
+          "value": 42.000,
+          "unit": "umol/L"
+        }
+      }
+    },
```

Comment:
Apologies for the reviewer - I have formatted the bundle which has introduced a bunch of formatting changes which skewered the changes, just wanted to highlight this was added.